### PR TITLE
wid: Add support for GAP WID 403

### PR DESCRIPTION
**Pull request by sjanc**
This was affecting GAP/CONN/DCEP/BV-05-C and GAP/CONN/DCEP/BV-06-C.

### CODE REVIEW

**Review by michal-narajowski — APPROVED** [review]

File: wid/gap.py [hunk 1]
```diff
@@ -954,6 +954,18 @@ def hdl_wid_402(desc):
     return True
 
 
+def hdl_wid_402(desc):
```
Comment (michal-narajowski) [inline wid/gap.py:957]: Did you mean 403?

**Review by michal-narajowski — CHANGES_REQUESTED** [review]

File: wid/gap.py [hunk 2]
```diff
@@ -954,6 +954,18 @@ def hdl_wid_402(desc):
...
+    #Please perform the Directed Connection Establishment Procedure using RPA
+    #then resolve the PTS address and connect with PTS.
+    stack = get_stack()
+
+    if not stack.gap.iut_has_privacy():
+        return False
+
+    btp.gap_conn()
```
Comment (michal-narajowski) [inline wid/gap.py:965]: I think this does not test DCEP. Under the hood, this sends a Gap Connect command to IUT with PTS address as an argument.

File: wid/gap.py (hunk 1 above)
Comment (michal-narajowski) [reply wid/gap.py:957]: Might make sense to call 402 if the expected behavior is the same.

Comment (sjanc) [reply wid/gap.py:957]: yes, it was suppose to be 403 :)

Comment (sjanc) [reply wid/gap.py]: one is general, and one is directed,  I'd prefer to keep those separate even if currently implementations match

**Review by michal-narajowski — APPROVED** [review]

**Pull request merged**